[Doc]: function imports (#503)
SAP/cloud-sdk: docs/js/features/odata/odata-v2-client.mdx

  .execute(destination);
```

## Function imports
### Request builder

> The [function imports request builder](pathname:///api/1.28.1/classes/sap_cloud_sdk_core.functionimportrequestbuilderv2) helps build a request for a service operation with essential parameters.

Reviewer: ```suggestion
The [function imports request builder](pathname:///api/1.28.1/classes/sap_cloud_sdk_core.functionimportrequestbuilderv2) helps build a request for a service operation containing parameters in a typesave way.
```

The example below creates a function import request builder for the service operation "PostGoodsIssue" and then execute it against your service.
```ts
postGoodsIssue({outboundDeliveryOrder: "order"})
  .execute(destination);
```

> The service operation is defined in the service metadata. Therefore, for some services, there might be no function imports at all like the beloved `Business Partner Service`.

Reviewer: I would swapt the order. Fist say where the `PostGoddIssue` is loacated (the warehou servie). And them make something like:

`Note that not all OData service contain function imports. For example the BP service does not. bla bla bla`. I also would remove the beloved.
